decred_wallet/view_controller/SendViewController.swift

```diff
@@ -229,40 +227,40 @@ class SendViewController: UIViewController, UITextFieldDelegate,UITextPasteDeleg
     }
     
     func removePasteBtn(){
-         DispatchQueue.main.async {
+        DispatchQueue.main.async {
             if !(self.removedBtn){
-            
-            self.toAddressContainer.removeArrangedSubview(self.pasteBtn)
-            self.pasteBtn.isHidden = true
+                
+                self.toAddressContainer.removeArrangedSubview(self.pasteBtn)
+                self.pasteBtn.isHidden = true
                 //self.removedBtn = true
```

Comment:
Commented lines of code and empty white lines can be removed from this if/else blocks.